eclipse-collections/src/main/java/org/eclipse/collections/impl/bag/immutable/ImmutableArrayBag.java

```diff
@@ -246,6 +246,25 @@ public T getLast()
     }
 
     @Override
+    public T getOnly()
+    {
+        Iterator<T> iterator = this.iterator();
+
```

Comment:
ImmutableArrayBag extends AbstractRichIterable, so it already has an implementation of getOnly(), as most collections do. If we're going to override that implementation, it should be to avoid the use of an iterator. ImmutableArrayBag.getFirst() doesn't use an iterator, so we can probably avoid it here too.